doc-content/enterprise-only/installation/patches-applying-proc.adoc

```diff
@@ -109,15 +109,6 @@ The update script creates a `backup` folder in the extracted `{PRODUCT_INIT}-$VE
 . After the update tool completes, return to the *Software Downloads* page of the Red Hat Customer Portal where you downloaded the update tool and install any other updates or new release artifacts that are relevant to your {PRODUCT} distribution.
 +
 For files that already exist in your {PRODUCT} distribution, such as `.jar` files for the {DECISION_ENGINE} or other add-ons, replace the existing version of the file with the new version from the Red Hat Customer Portal.
-. If you use the standalone *{PRODUCT} {PRODUCT_VERSION_LONG} Maven Repository* artifact (`{PRODUCT_FILE}-maven-repository.zip`), such as in air-gap environments, download *{PRODUCT} [VERSION] Incremental Maven Repository* and extract the downloaded `{PRODUCT_INIT}-$VERSION-incremental-maven-repository.zip` file to your existing `~/maven-repository` directory to update the relevant contents.
-+
---
-Example Maven repository update:
-[source,subs="attributes+"]
-----
-$ unzip -o {PRODUCT_INIT}-{PRODUCT_VERSION}.1-incremental-maven-repository.zip -d $REPO_PATH/{PRODUCT_INIT}-{PRODUCT_VERSION}.0-maven-repository/maven-repository/
```

Comment:
Please just change to the full repository zip instead of the incremental repository zip.

Reply:
Sorry, I have just realized that the full repository zip contains two top-level folders which we do not want to be recreated in the target Maven repository. Therefore, the unzip command needs to be changed to the following two commands (it seems there is no way how to accomplish this with just parameterizing the unzip command):

```
$ unzip -o {PRODUCT_INIT}-{PRODUCT_VERSION}.1-maven-repository.zip 'rhba-{PRODUCT_VERSION}.1.GA-maven-repository/maven-repository/*' -d /tmp/rhbaMavenRepoUpdate 
$ mv /tmp/rhbaMavenRepoUpdate/rhba-{PRODUCT_VERSION}.1.GA-maven-repository/maven-repository/ $REPO_PATH/{PRODUCT_INIT}-{PRODUCT_VERSION}.0.GA-maven-repository/maven-repository/
```

Reply:
The `/tmp/rhbaMavenRepoUpdate` folder may then be deleted.